<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adpartment</class>
 <widget class="QDialog" name="adpartment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>657</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/new/adpartment 2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                       color:white;
     background-color: transparent;
                       }
                         QPushButton:hover {
                             background-color: gray;
                     		 border-radius: 20px;


                         }

QPushButton:pressed
 {

      background-color : rgb(51, 149, 255);
      

 }</string>
   </property>
   <property name="text">
    <string>&gt;Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>127</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>ay hagaa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>182</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>52252</string>
   </property>
  </widget>
  <widget class="QLabel" name="location">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>225</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>5555</string>
   </property>
  </widget>
  <widget class="QLabel" name="area">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>135</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>5555</string>
   </property>
  </widget>
  <widget class="QLabel" name="numBed">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>182</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>55555</string>
   </property>
  </widget>
  <widget class="QLabel" name="numBathrooms">
   <property name="geometry">
    <rect>
     <x>685</x>
     <y>225</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>5555</string>
   </property>
  </widget>
  <widget class="QLabel" name="descrip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                       color:white;
background-color: transparent;
                     		 border-radius: 20px;                       }
                         QPushButton:hover {
                             background-color: #2E2E2E;
                     		 border-radius: 20px;


                         }

QPushButton:pressed
 {

      background-color : rgb(51, 149, 255);
      

 }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/3d-cartoon-portrait-person-practicing-law-related-profession.png</normaloff>:/new/3d-cartoon-portrait-person-practicing-law-related-profession.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>230</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>hany mostafa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="edit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                       color:white;
     background-color: #2E2E2E;
                     		 border-radius: 20px;

                       }
                         QPushButton:hover {
                             background-color: gray;
                     		 border-radius: 20px;


                         }

QPushButton:pressed
 {

      background-color : rgb(51, 149, 255);
      

 }</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                       color:white;
     background-color: #2E2E2E;
                     		 border-radius: 20px;

                       }
                         QPushButton:hover {
                             background-color: gray;
                     		 border-radius: 20px;


                         }

QPushButton:pressed
 {

      background-color : rgb(51, 149, 255);
      

 }</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="type">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>86</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>13</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}&lt;?xml version=&quot;1.0&quot; encoding=&quot;UTF-8&quot;?&gt;
&lt;ui version=&quot;4.0&quot;&gt;
 &lt;widget name=&quot;__qt_fake_top_level&quot;&gt;
  &lt;widget class=&quot;QLabel&quot; name=&quot;name_2&quot;&gt;
   &lt;property name=&quot;geometry&quot;&gt;
    &lt;rect&gt;
     &lt;x&gt;300&lt;/x&gt;
     &lt;y&gt;210&lt;/y&gt;
     &lt;width&gt;63&lt;/width&gt;
     &lt;height&gt;20&lt;/height&gt;
    &lt;/rect&gt;
   &lt;/property&gt;
   &lt;property name=&quot;font&quot;&gt;
    &lt;font&gt;
     &lt;family&gt;Segoe UI Black&lt;/family&gt;
     &lt;pointsize&gt;15&lt;/pointsize&gt;
     &lt;bold&gt;true&lt;/bold&gt;
    &lt;/font&gt;
   &lt;/property&gt;
   &lt;property name=&quot;text&quot;&gt;
    &lt;string&gt;TextLabel&lt;/string&gt;
   &lt;/property&gt;
  &lt;/widget&gt;
 &lt;/widget&gt;
 &lt;resources/&gt;
&lt;/ui&gt;
</string>
   </property>
   <property name="text">
    <string>hometown</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
                 
     background-color: black;
 	border-radius: 20px;

}
QPushButton:hover
{
 border-radius: 20px;
background-color : gray;
 icon-size: 16px,16px;

}
QPushButton:pressed
 {

      background-color : rgb(51, 149, 255);
       icon-size: 25px,25px;


 }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/new/0684456b-aa2b-4631-86f7-93ceaf33303c.png</normaloff>:/new/0684456b-aa2b-4631-86f7-93ceaf33303c.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="images">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>360</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
